doc/changes/DM-44156.misc.rst

```diff
@@ -0,0 +1 @@
+Added a global default values for ``memoryLimit`` to let the users use the BPS automatic memory scaling mechanism.
```

Comment:
If the plan is to remove memoryLimit from ctrl_bps once DM-44110 is complete, I would mention that here.

Reply:
Also should be singular value.   And it's using the mechanism with allocateNodes?

Reply:
At the moment, I have no plans to remove it once DM-44110 is completed. It's a fairly generic BPS parameter and can be easily overridden if needed considering the priority order
DM-44110 will provide.